_config.yml

```diff
@@ -18,6 +18,16 @@ description: Primer Spec is a Jekyll theme for GitHub Pages
 primerSpec:
   sitemap:
     label: Docs & Demos
+    externalLinks:
+      - title: "Google" 
+        url: https://google.com/
+      - title: "EECS 485"
+        url: https://eecs485.org
+    order:
```

Comment:
I worry that with this API, it’s easy for this config’s list to go out-of-date with the other pages. I also don’t like that website admins need to type the page title in this config and hope they didn’t make a typo. 

What do you think about adding a `sitemap-order` attribute to each page’s front matter, and then use that to determine order? Almost like z-indexes or @awdeorio’s scheme for naming image files 😆 
This way, the order of the page lives along with the rest of the page’s metadata. 

I’m open to other suggestions for defining the order :)

Reply:
I worry that with this API, it’s easy for this config’s list to go out-of-date with the other pages. I also don’t like that website admins need to type the page title in this config and hope they didn’t make a typo.

That's a good point. I think there's a trade-off between storing the order with individual page data and storing it in a central location though. If the order is determined by each page's front matter, then you have to look at each other page's front-matter in order to rearrange the order. Having the location of each page in a central location makes it easier to see each page's location relative to the others. Also, even if the maintainers make a typo or forget to make a change (or add the location of a new page), the page will still appear in the sidebar: it just won't be in the right order. If the order really matters, it would be easy to fix a mistake like that, right?